azure/postgres.sh

```diff
@@ -69,7 +69,7 @@ fn azure_postgres_set_max_size(instance, size) {
 # azure_postgres_set_version sets the Postgres version of a server.
 # `instance` is the postgres server instance.
 # `version` is the postgres server version.
-# Default: 9.5
+# Default: latest
```

Comment:
Yes this is WIP. But do you think it's a big deal to change this? Is just that Azure changed their default version and only now I realized they will always be using the latest version.